### PR TITLE
porting ltp openposix testsuites

### PR DESCRIPTION
**Pull request by ftjahjad**

### CODE REVIEW

**Comment by ftjahjad** [comment]
Hi @rasibley , and @veruu 

Please review the PR.

Thanks.

File: distribution/ltp/openposix_testsuite/patches/20140115/run-tests.sh [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+	return $ret_code
+
+}
+
+# SETUP
+if [ -w "$LOGFILE" ] || echo "" > "$LOGFILE"; then
+	:
+else
+	echo >&2 "ERROR: $LOGFILE not writable"
+	exit 1
```
Comment (rasibley) [inline distribution/ltp/openposix_testsuite/patches/20140115/run-tests.sh]: hi @ftjahjad should we abort in this case ? This looks like a setup issue.

File: distribution/ltp/openposix_testsuite/patches/20140115/run-tests.sh [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+fi
+
+SCRIPT_DIR=$(dirname "$0")
+TEST_PATH=$1; shift
+T0=$SCRIPT_DIR/t0
+T0_VAL=$SCRIPT_DIR/t0.val
+
+if [ ! -x $T0 ]; then
+	echo >&2 "ERROR: $T0 doesn't exist / isn't executable"
+	exit 1
```
Comment (rasibley) [inline distribution/ltp/openposix_testsuite/patches/20140115/run-tests.sh]: Same here, should we add an abort ?

File: distribution/ltp/openposix_testsuite/patches/20140422/run-tests.sh [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+	return $ret_code
+
+}
+
+# SETUP
+if [ -w "$LOGFILE" ] || echo "" > "$LOGFILE"; then
+	:
+else
+	echo >&2 "ERROR: $LOGFILE not writable"
+	exit 1
```
Comment (rasibley) [inline distribution/ltp/openposix_testsuite/patches/20140422/run-tests.sh]: Same here, add an abort here if logfile isn't present ?

File: distribution/ltp/openposix_testsuite/patches/20140422/run-tests.sh [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+fi
+
+SCRIPT_DIR=$(dirname "$0")
+TEST_PATH=$1; shift
+T0=$SCRIPT_DIR/t0
+T0_VAL=$SCRIPT_DIR/t0.val
+
+if [ ! -x $T0 ]; then
+	echo >&2 "ERROR: $T0 doesn't exist / isn't executable"
+	exit 1
```
Comment (rasibley) [inline distribution/ltp/openposix_testsuite/patches/20140422/run-tests.sh]: Same as above

File: distribution/ltp/openposix_testsuite/patches/20140115/run-tests.sh (hunk 2 above)
Comment (ftjahjad) [reply distribution/ltp/openposix_testsuite/patches/20140115/run-tests.sh]: we should add an abort :100:

File: distribution/ltp/openposix_testsuite/patches/20140422/run-tests.sh (hunk 3 above)
Comment (ftjahjad) [reply distribution/ltp/openposix_testsuite/patches/20140422/run-tests.sh]: Yes again, we should add an abort

File: distribution/ltp/openposix_testsuite/patches/20140422/run-tests.sh (hunk 4 above)
Comment (ftjahjad) [reply distribution/ltp/openposix_testsuite/patches/20140422/run-tests.sh]: yes again, need an  abort

**Comment by ftjahjad** [comment]
@rasibley ,
Please review again with additional abort/warn

**Comment by rasibley** [comment]
@major can you review ? @ftjahjad has example beaker jobs (passing) in the FMK ticket

**Comment by ftjahjad** [comment]
@rasibley , thanks for reviewing

**Pull request merged**